content/chainguard/migration-guides/migrating-to-php-images.md

```diff
@@ -0,0 +1,187 @@
+docker run -it --rm --entrypoint /bin/sh --user root cgr.dev/chainguard/php:latest-dev
+```
+
+Make sure to update the package manager cache:
+
+```shell
+apk update
+```
+If you want to search for PHP 8.2 XML extensions, for example, you can run the following:
+
+```shell
+apk search php*8.2*xml*
+```
+And this should give you a list of all PHP 8.2 XML extensions available in Wolfi.
+
+```
+php-8.2-simplexml-8.2.17-r0
+php-8.2-simplexml-config-8.2.17-r0
+php-8.2-xml-8.2.17-r0
+php-8.2-xml-config-8.2.17-r0
+php-8.2-xmlreader-8.2.17-r0
+php-8.2-xmlreader-config-8.2.17-r0
+php-8.2-xmlwriter-8.2.17-r0
+php-8.2-xmlwriter-config-8.2.17-r0
+php-simplexml-8.2.11-r1
+php-xml-8.2.11-r1
+php-xmlreader-8.2.11-r1
+php-xmlwriter-8.2.11-r1
+```
+For more searching tips, check the [Searching for Packages](/chainguard/migration-guides/migrating-to-chainguard-images/#searching-for-packages) section of our base migration guide.
+
+## Migrating PHP CLI workloads to use Chainguard Images
+Our `latest` and `latest-dev` PHP image variants are designed to run CLI applications and scripts that don't need a web server. As a first step towards migration, you might want to change your base image to the `latest-dev` variant, since that would be the closest option for a drop-in base image replacement. Once you have your dependencies and steps dialed in, you can optionally migrate to a multi stage build to create a strict runtime containing only what the application needs to run.
+
+The following `Dockerfile` uses the `php:latest-dev` image to build the application, which in this case means copying the application files and installing dependencies via `composer`. A second build stage copies the application to a final distroless image based on `php:latest`.
+
+```Dockerfile
+FROM cgr.dev/chainguard/php:latest-dev AS builder
+USER root
+COPY . /app
+RUN chown -R php /app
+USER php
+RUN cd /app && \
+    composer install --no-progress --no-dev --prefer-dist
+
+FROM cgr.dev/chainguard/php:latest
+COPY --from=builder /app /app
+
+ENTRYPOINT [ "php", "/app/myscript.php" ]
+```
+
+Our [PHP Getting Started](/chainguard/chainguard-images/getting-started/php/) guide has step-by-step instructions on how to build and run a PHP CLI application with Chainguard Images.
+
+## Migrating PHP Web applications to use Chainguard Images
+For PHP web applications that serve content through a web server, you should use the `latest-fpm` and `latest-fpm-dev` variants of our PHP image. Combine it with our [Nginx image](/chainguard/chainguard-images/reference/nginx) and an optional database for a traditional LEMP setup.
+
+The overall migration process is essentially the same as described in the previous section, with the difference that you won't set up application entry points, since these images run as services. Your Dockerfile may require additional steps to set up front end dependencies, initialize databases, and perform any additional tasks needed for the application to run through a web server.
```

Comment:
```suggestion
The overall migration process is essentially the same as described in the previous section, with the difference that you won't set up application entry points, since these images run as services. Your Dockerfile may require additional steps to set up front-end dependencies, initialize databases, and perform any additional tasks needed for the application to run through a web server.
```

based mostly on [this](https://medium.com/@ist.stevkovski/is-it-front-end-or-front-end-or-frontend-3ae717cae4aa) which is not a reliable grammar reference but it looks right to me